lib/fog/aws/models/elb/load_balancer.rb

```diff
@@ -223,16 +223,24 @@ def save
         end
 
         def reload
-          super
           @instance_health = nil
           @policy_descriptions = nil
-          self
+          super
         end
 
         def destroy
           requires :id
           service.delete_load_balancer(id)
         end
+
+        protected
+
+        def all_associations_and_attributes
+          super.merge(
+            'ListenerDescriptions' => attributes['ListenerDescriptions'],
+            'BackendServerDescriptions' => attributes['BackendServerDescriptions'],
+          )
+        end
```

Comment:
Why do you need this? Maybe we need to improve this method on `fog-core`?

Reply:
Attributes are not registered in by an `attribute` declaration but are used to collections from the underlying attributes hash.

This seems like a reasonable workaround using normal OO methodology.

Reply:
Got it. I'm not really pointing this as a problem but i'm more interested in knowing the reasons behind it. If it does inflect an in improvement in `fog-core` i would be more then happy to do it myself. =)

Are those attributes not being used by the DSL for a particular reason? Is there something i can improve in `fog-core` to help improve the usage?

Reply:
Hmmm. So you mean an `attribute` that ideally does not belong to this resource but the API expects? Like a concept issue from the API standpoint?

BTW, this is not a blocker. I'm just trying to figure out the use case to see if/how we can improve `fog-core`. My main goal now is to get it to a better state.